src/Microsoft.Data/DataFrame.cs

```diff
@@ -92,5 +111,10 @@ public IList<IList<object>> Tail(int numberOfRows)
         }
         // TODO: Add strongly typed versions of these APIs
         #endregion
+
+        public override bool Equals(object obj)
+        {
+            return ReferenceEquals(this, obj);
```

Comment:
I don't believe this is necessary. This Equals override can be removed.

Reply:
Deleted